### DISCUSSION TITLE
How do you intercept everything from a virtual machine?

### DISCUSSION THREAD

**Discussion by GEOEGII555** [mitmproxy/mitmproxy#7024] [Q&A]
How can I make mitmproxy intercept everything that's on a VM without installing anything (other than the certificate authority certificate) on the VM itself, only the host machine?

**Comment by emanuele-em** [comment]
There is well written documentation exactly on what you need 🙂 
https://docs.mitmproxy.org/stable/howto-transparent-vms/

**Reply by GEOEGII555** [reply]
I'll check that out, thanks.

Although I wonder what if both the guest and the host machines run Windows?
